### DISCUSSION TITLE
<GatsbyImage /> border-radius not working on safari

### DISCUSSION THREAD

**Discussion by Soneji** [gatsbyjs/gatsby#30447] [Help] [answered]
Hi

I am using `<GatsbyImage />` to load in a blurred [background image](https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image#background-images) and then it loads the proper image. The problem I am having is the blurred image shows the border-radius but when the full image loads, it does not have the border radius.

See 
- [Source](https://github.com/portfolio)
- [file from above source](https://github.com/soneji/portfolio/blob/main/src/components/header.js)
- [borderRadius style from above file](https://github.com/soneji/portfolio/blob/b0ad0fce59643d3ebf19ba77055893bcd6f43b06/src/components/header.js#L72) 
- [live site](https://dhavalsoneji.com)

This is only happening in Safari (desktop and mobile) and works fine on Chrome+Firefox

**Comment by rodneylab** [comment, marked as answer]
Hi Dhaval,

Did you work this out?

I was seeing something similar in Safari on my site.  I added am imgStyle attribute to GatsbyImage and that seemed to fix it.  For your code, you would need to try:

```jsx
                                        <GatsbyImage
                                            style={{
                                                gridArea: "1/1",
                                                backgroundSize: "cover",
                                                width: "200px",
                                                height: "200px",
                                                borderRadius: "20px",
                                            }}
                                            imgStyle={{ borderRadius: "20px" }}
                                            layout="fullWidth"
                                            alt={"Picture of " + data.name}
                                            image={getImage(data.avatarFile)}
                                            formats={["auto", "webp", "avif"]}
                                        />
```

Keep the other style attribute as it is as that might still be needed by other browsers.

Let me know if that works,

Rodney

**Reply by Soneji** [reply]
Hey Rodney! This worked, thank you so much! :D 
You're a legend
